site/app/templates/autograding/TAResults.twig

```diff
@@ -28,16 +28,38 @@
         </div>
         {% if any_late_days_used %}
             <a name="show_late_table" style="float:right" class="btn btn-primary" href="{{ late_days_url }}">Show my late days information</a>
+            <br />
```

Comment:
Instead of 2 `<br />` tags, you should add styling like `padding-bottom: 2em` or `margin-bottom: 2em`

Reply:
Thanks for looking! I'll get on that :)